ddev/ddev
# fix: Install matching postgresql client versions in ddev-webserver, fixes #6139

## The Issue

* #6139 

## How This PR Solves The Issue

This is the (hopefully) last of a series of PRs that attempts to put the matching version of database client in the running web container as well as the db container. 

This one is about PostgreSQL, the clients `psql`, `pg_dump`, and `pg_restore`

This works for postgres 10+. For Postgres 9, it's not in the provided repository, so we use the default postgres clients

Note that this changes the *default* psql in the web container from 15 to 16. However, that shouldn't matter, because if someone is using Postgres, the version will match (unless postgres:9)

## Manual Testing Instructions

Start a project with a database of type postgres. `ddev psql --version` should show that version.

## Automated Testing Overview

Added TestWebserverPostgresDBClient

## Release/Deployment Notes
